_test/test_change_crystal/test_change_crystal_bragg_coarse.m

```diff
@@ -245,7 +245,7 @@ function test_bragg_pos(obj)
             assertElementsAlmostEqual(width,width_sample,'absolute',1.e-1);
             % contains path, so need no-string comparison
             assertEqualToTolWithSave(obj,wcut,1.e-6,'ignore_str',true);
-            assertEqualToTolWithSave(obj,wpeak,1.e-6,'ignore_str',true);
+            assertEqualToTolWithSave(obj,wpeak,[1.e-3,1.e-5],'ignore_str',true);
```

Comment:
worth commenting why these sizes. Presuming they are - get an approximate equality for the test, sufficiently narrow to accept the result - but we should make it clear

Reply:
This is still draft, I'm trying to work out myself why the accuracy has changed and how far off they are